dj/sql/parsing/backends/sqloxide.py

```diff
@@ -434,14 +421,14 @@ def parse_oxide_tree(parse_tree: dict) -> Query:
     raise DJParseException("Failed to parse query")  # pragma: no cover
 
 
-def parse(sql: str) -> Query:
+def parse(sql: str, dialect: str = "hive") -> Query:
```

Comment:
Let's keep "ansi" as default?
```suggestion
def parse(sql: str, dialect: str = "ansi") -> Query:
```

Reply:
+1